cmd/kubectl-gadget/trace/capabilities.go

```diff
@@ -59,12 +63,23 @@ func newCapabilitiesCmd() *cobra.Command {
 				name:        "capabilities",
 				commonFlags: commonFlags,
 				parser:      NewCapabilitiesParser(&commonFlags.OutputConfig),
+				params: map[string]string{
+					types.FilterNonAuditChecksParam: strconv.FormatBool(filterNonAuditChecks),
+				},
 			}
 
 			return capabilitiesGadget.Run()
 		},
 	}
 
+	cmd.PersistentFlags().BoolVarP(
+		&filterNonAuditChecks,
+		"filter-non-audit-checks",
```

Comment:
What about shorter names like `only-audit`, `only-audit-checks`? (I personally find it difficult to understand the "filter + non" combination.

Reply:
Updated to `--only-audit`